arch/arm64/boot/dts/freescale/imx93-charge-som.dtsi

```diff
@@ -57,6 +57,26 @@
 	pinctrl-1 = <&pinctrl_lpi2c3_gpio>;
 	scl-gpios = <&gpio2 29 (GPIO_ACTIVE_HIGH | GPIO_OPEN_DRAIN)>;
 	sda-gpios = <&gpio2 28 (GPIO_ACTIVE_HIGH | GPIO_OPEN_DRAIN)>;
+
+	eeprom@50 {
+		nvmem-layout {
+			compatible = "fixed-layout";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			eth1_host_mac_address: eth1_host_mac_address@100 {
```

Comment:
- node ref is fine
- underscore should be avoided for node names, please use dashes instead
- also the node name should represent the general function so how about "mac" or "base-mac-address" like in the other dts files